Models/ContainerBuilder.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Infrastructure\Models;
+
+class ContainerBuilder extends \Symfony\Component\DependencyInjection\ContainerBuilder
+{
+    public function register($id, $class = null, $public = true)
```

Comment:
I'd prefer just add an additional method, for example, addShared in Phalcon or something similar depends on purpose

Reply:
Because of symfony, all services in container if it's not compiled is in two states in same time: public and private.

And only because we had a lick in our application we didn't compile container. I just made obvious that how it behaves. And extended current container. 
This makes my changes compatible and require less changes. More over we don't use full features from symfony in our framework and container in our app is isolated, we don't do autowire.

Reply:
as containers are isolated and we can have alist of continers it doesn't make seanse to follow symfony/phalcon.

We don't have private or public in prespective of symfony how the see it.